src/config.js

```diff
@@ -88,6 +89,7 @@ const config = {
   ltiActivated,
   ndlaPersonalClientId: getEnvironmentVariabel('NDLA_PERSONAL_CLIENT_ID', ''),
   auth0Domain: getEnvironmentVariabel('AUTH0_DOMAIN', ''),
+  auth0Api: 'https://ndla.eu.auth0.com/api/v2/',
```

Comment:
Why is this a config (when it is static)? Also https://ndla.eu.auth0.com/ should be used for both token and users in auth.js